caiman_napari/main_offline_gui.py

```diff
@@ -186,14 +194,18 @@ def show_cnmf_params_gui(self):
     def show_mcorr_params_gui(self):
         self.mcorr_gui = MCORRWidget(parent=self)
         self.mcorr_gui.show()
-    def show_cnmf_results(self):
+    def load_output(self):
         print("show cnmf results")
         item_gui = QtWidgets.QListWidgetItem = self.ui.listWidgetItems.currentItem()
         uuid = item_gui.data(3)
-        dir = os.path.dirname(self.dataframe_file_path)
-        self.cnmf_obj = load_CNMF(dir + '/' + uuid + '.hdf5')
-
-        show_results(self.cnmf_obj, self.viewer)
+        ix = self.dataframe[self.dataframe['uuid'] == uuid].index[0]
+        algo = self.dataframe['algo'][ix]
+        self.viewer.open(self.dataframe['input_movie_path'][ix])
+        print("algo name", algo)
+        if algo == 'cnmf':
```

Comment:
After organizing the `_cnmf` etc. modules into a new `algorithms` module we can just do this which is much better:

```python
from . import algorithms

algo = self.dataframe['algo'][ix]
getattr(algorithms, algo).load_output(self.viewer, self.dataframe.iloc[ix])()
```
`getattr` will dynamically load in a universal manner, don't need to manually code in everything!